tests/post/steps/test_service_configuration.py

```diff
@@ -92,30 +78,26 @@ def update_service_configuration(
     value
 ):
 
-    csc_from_yaml = yaml.safe_load(
-        DEFAULT_CSC_CONFIG_YAML.format(replicas=value)
-    )
+    csc_obj = csc.get(name, namespace)
+    csc_content = csc.load(csc_obj, name, namespace)
+
+    current = csc_content
+    elements = path.split('.')
+    for element in elements[:-1]:
+        current = current.setdefault(element, {})
+    current[elements[-1]] = value
```

Comment:
don't we have this defined in a helper (`utils.set_dict_element` or something) already?

Reply:
Yes, but it'll not work if the key does not already exist.

Reply:
Maybe it should be fixed?

Reply:
Maybe, didn't know if it was intended or not... but since this function is not used anywhere, I'll change it;